index.html

```diff
@@ -151,18 +151,21 @@
     </p>
 
     <p>
-<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to
-provide verifiable, decentralized digital identity. These new identifiers are
-designed to enable the controller of a <a>DID</a> to prove control over
-it and to be implemented independently of any centralized registry, identity
-provider, or certificate authority. <a>DIDs</a> are URLs that relate a
-<a>DID subject</a> to a <a>DID document</a> allowing trustable interactions with
-that subject. <a>DID documents</a> are simple documents describing how to use
-that specific <a>DID</a>. Each <a>DID document</a> can express cryptographic
-material, verification methods, or <a>service endpoints</a>, which provide a
-set of mechanisms enabling a <a>DID controller</a> to prove control of the
-<a>DID</a>. <a>Service endpoints</a> enable trusted interactions with the
-<a>DID subject</a>.
+<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to 
+provide verifiable, decentralized digital identity. A <a>DID</a> may identify 
+any subject (e.g., a person, organisation, thing, datamodel, abstract entity, etc.)
+that the controller of that <a>DID</a> decides that it 
+identifies. These new identifiers are designed to enable the controller 
+of a <a>DID</a> to prove control over it and to be implemented independently 
+of any centralized registry, identity provider, or certificate authority. 
+<a>DID</a>s are URLs that associate a <a>DID subject</a> with a 
+<a>DID document</a> allowing trustable interactions associated with that subject. 
+Each <a>DID document</a> can express cryptographic material, verification methods, 
+or <a>service endpoints</a>, which provide a set of mechanisms enabling a 
+<a>DID controller</a> to prove control of the <a>DID</a>. <a>Service 
+endpoints</a> enable trusted interactions associated with the <a>DID 
+subject</a>. A <a>DID document</a> may contain semantics about the subject 
+that its identifies, or even the <a>DID subject</a> itself (e.g. a data model).
```

Comment:
I don't think this is right, but I'm also not clear what it means. 
>A <a>DID document</a> might contain semantics about the subject that it[s] identifies, or even the <a>DID subject</a> itself (e.g. a data model).

This seems both redundant--both parts of the sentence are saying the same thing to my read--and I think there is an anti-pattern implied herein. IMO, DID Documents should NOT contain semantic information about the Subject, only about how to securely interact with that subject. This is not a directory architecture.

Reply:
Just wanted to note that https://github.com/w3c/did-core/issues/199#issuecomment-589401616 is an example of a DID document containing the subject - not just information about the subject, but actually the subject itself.

I am not endorsing this model, simply pointing to it.  This is an example of "the DID subject itself" as different from "about the subject".

Reply:
Ahh... I read "about" as referring to "the subject" or "even the subject it identifies."

The two phrases combined with that "or even" are not grammatically symmetric.

Reply:
I separated two two sentences, to prevent this misreading, see
https://github.com/w3c/did-core/pull/213/commits/2d51aa7b069822abf82f59ef410e653f94052502
Does that help?